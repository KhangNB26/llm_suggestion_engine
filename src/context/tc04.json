{
  "workItems": [
    {
      "work_id": "11111111-aaaa-bbbb-cccc-000000000003",
      "task_id": "11111111-aaaa-bbbb-cccc-000000000003",
      "title": "On tap Hoa hoc",
      "effective_deadline": "2025-09-26T15:00:00Z",
      "effective_priority": 1,
      "status": 2,
      "is_atomic": true,
      "task_deadline": "2025-09-26T15:00:00Z",
      "task_priority": 1,
      "task_status": 2
    }
  ],
  "scheduleEntries": [],
  "freeSlots": [],
  "studyProfile": {
    "user_id": "11111111-1111-1111-1111-111111111111",
    "chronotype": "early_bird",
    "focusStyle": "deep_work",
    "workStyle": "solo"
  }
}
