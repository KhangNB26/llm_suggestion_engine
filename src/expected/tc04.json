{
  "items": [],
  "confidence": 0,
  "reason": "Khong co cong viec dang mo hoac khong co slot trong."
}
